src/NuGet.Clients/NuGet.MSSigning.Extensions/NuGet.MSSigning.Extensions.csproj

```diff
@@ -11,10 +11,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <Reference Include="NuGet">
```

Comment:
why does it need to depend on the actual .exe? we  don't recommend people do that

Reply:
Also, do you really need the Reference includes for system, system.core on SDK-based csproj? (I know it's from an old check-in)